cmd/admin-info.go

```diff
@@ -120,49 +122,78 @@ func (u infoMessage) String() (msg string) {
 
 	// When service is offline
 	if u.Service == "off" {
-		msg += fmt.Sprintf("%s  %s\n", console.Colorize("InfoFail", dot), u.Addr)
-		msg += fmt.Sprintf("   Uptime : Server is %s", console.Colorize("InfoFail", "offline"))
+		msg += fmt.Sprintf("%s  %s\n", dot, console.Colorize("PrintB", u.Addr))
+		msg += fmt.Sprintf("  Status : %s", console.Colorize("InfoFail", "offline"))
 		return
 	}
 
 	// Print error if any and exit
 	if u.Err != "" {
-		msg += fmt.Sprintf("%s  %s\n", console.Colorize("InfoFail", dot), u.Addr)
-		msg += fmt.Sprintf("   Uptime : Server is %s\n", console.Colorize("InfoFail", "offline"))
-		msg += fmt.Sprintf("    Error : %s", u.Err)
+		msg += fmt.Sprintf("%s  %s\n", dot, console.Colorize("PrintB", u.Addr))
+		msg += fmt.Sprintf("   Status : %s\n", console.Colorize("InfoFail", "offline"))
+		e := u.Err
+		if strings.Trim(e, " ") == "rpc: retry error" {
```

Comment:
How about we do this on server.. ?? Because if server changes the error this will fail..

Reply:
Short term fix. I can also add this on the server.